src/installer/publish/prepare-artifacts.proj

```diff
@@ -7,10 +7,13 @@
     <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
   </PropertyGroup>
 
+  <!-- Create all the checksum files. Exclude anything that ends
+       in .nupkg (symbol and regular packages). -->
   <Target Name="CreateChecksums">
     <ItemGroup>
       <ArtifactsForGeneratingChecksums
         Include="@(UploadToBlobStorageFile)"
+        Excldude="*.nupkg"
```

Comment:
There is a typo in `Exclude`.
```suggestion
        Exclude="*.nupkg"
```

Reply:
Should we include *.snupkg so that when/if we add them, they're excluded?

Reply:
They won't be found by any of this by default, so nothing needs to be done to prevent that:

https://github.com/dotnet/runtime/blob/188243a1ec2abb19f5d37d8274afa1994e848dd2/src/installer/publish/Directory.Build.targets#L21-L24

FWIW, I have concerns that snupkg doesn't meet dotnet/runtime requirements based on the spec last time I looked: https://github.com/dotnet/arcade/issues/167#issuecomment-478787936.